From 2025b53de6b3d97285d7c5f80497493007c586c3 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 30 Jan 2023 11:27:07 -0800
Subject: [PATCH] Fix build with gcc 13 by including <cstdint>

gcc 13 moved some includes around and as a result <cstdint> is
no longer transitively included [1]. Explicitly include it for
int32_t.

[1] https://gcc.gnu.org/gcc-13/porting_to.html#header-dep-changes

Upstream-Status: Submitted [https://github.com/tesseract-ocr/tesseract/pull/4009]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/ccutil/params.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ccutil/params.h b/src/ccutil/params.h
index f514d870..0f3f8743 100644
--- a/src/ccutil/params.h
+++ b/src/ccutil/params.h
@@ -21,6 +21,7 @@
 
 #include <tesseract/export.h> // for TESS_API
 
+#include <cstdint>
 #include <cstdio>
 #include <cstring>
 #include <string>
-- 
2.39.1

